src/components/StatusAlert/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { StatusAlert as Alert, Icon } from '@edx/paragon';
+
+import './StatusAlert.scss';
+
+const StatusAlert = (props) => {
+  const {
+    alertType,
+    className,
+    iconClassName,
+    title,
+    message,
+  } = props;
+
+  return (
+    <Alert
+      className={className}
+      alertType={alertType}
+      dismissible={false}
+      dialog={
+        <div className={
+          classNames({
+            'd-flex': iconClassName && iconClassName.length > 0,
```

Comment:
It should be fine to always be display: flex even if there's no icon. I would just set it in scss and remove this.